### ISSUE TITLE
Reproject using biquadratic and bicubic interpolation returns Nans

### ISSUE THREAD

**Issue by Sebokolodi** [astropy/reproject#464]
I am working on reprojecting SIN images to an HPX projection. While using nearest-neighbor and bilinear interpolation generates images successfully, these methods aren't optimal for conserving flux. I prefer to use biquadratic or bicubic interpolation, but unfortunately, all output images turn out as NaNs when I apply these methods. 

And there is no error message. There was similar issue before, but not sure if the issue was resolved.

**Comment by astrofrog** [comment]
Could you provide the input and output headers you are using?

**Comment by Sebokolodi** [comment]
[output_header.txt](https://github.com/user-attachments/files/16616676/output_header.txt)
[input_header.txt](https://github.com/user-attachments/files/16616678/input_header.txt)

**Comment by Sebokolodi** [comment]
Any update on this issue?

**Comment by astrofrog** [comment]
Yes I think that those interpolation techniques basically fail if there is a single NaN in the input image.
